app/javascript/mastodon/locales/zh-CN.json

```diff
@@ -159,22 +159,22 @@
   "privacy.public.short": "公共",
   "privacy.unlisted.long": "公开，但不在公共时间轴显示",
   "privacy.unlisted.short": "公开",
-  "relative_time.days": "{number}d",
-  "relative_time.hours": "{number}h",
-  "relative_time.just_now": "now",
-  "relative_time.minutes": "{number}m",
-  "relative_time.seconds": "{number}s",
+  "relative_time.days": "{number} 天",
+  "relative_time.hours": "{number} 小时",
```

Comment:
空格放在这里会造成数字和前面 at 粘在一起的错觉，干扰注意力。去掉吧。作为恶趣味的一种体现，可以考虑去掉”小“。

Reply:
Sure. Twitter isn't using spaces, but they do use `小时`; Weibo also have `前` (ago), should I add it?

Reply:
"ago" is implied, so I guess we won't need that. The 时 thing is really some "WE NEED TO GET TO ONE CHARACTER" and academic speaking thing. Well in 时's defense it's like using 分 instead of 分钟.

Reply:
Hmm, that makes sense. Just checked again, Twitter uses `分`... lol